docs/components/icon/README.md

```diff
@@ -101,7 +101,7 @@
                 "description": "Modifies the style variant for this component.  Possible values: { 'inherit-color' }",
                 "alert": {
                   "type": "updated",
-                  "description": "Updated in v2.0.0. See 'size' prop."
+                  "description": "Updated in v2.0.0. Moved sized values to 'size' prop."
```

Comment:
Also here... I like how you have the size description in button:  ```"Sets the button size; values can target responsive breakpoints. Breakpoint values are: xs, sm, md, and lg. Examples: { 'small' | 'medium' | 'large' | 'large@sm' }"```
Maybe also change this to be consistent?